src/qm_gen.pri

```diff
@@ -1,5 +1,5 @@
-TS_IN = $$fromfile(src/src.pro,TRANSLATIONS)
-TS_IN_NOEXT = $$replace(TS_IN,".ts","")
+TRANSLATIONS += $$files(lang/qbittorrent_*.ts)
```

Comment:
Why don't you use "src.pro" for extracting translation filenames?

Reply:
I suppose it is cleaner to place it together at lang/lang.pri?